<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>880</width>
      <height>740</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Home</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="PositionLabel_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>55</y>
        <width>660</width>
        <height>18</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="prevButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>55</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="messageLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>840</width>
        <height>600</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="enterButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>690</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_4"/>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Communities</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>370</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>AddCommunity</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>370</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>DeleteCommunity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>841</width>
        <height>281</height>
       </rect>
      </property>
      <property name="text">
       <string>CommunityList</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>370</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>FindCommunity</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Friends</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>821</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string>FriendsList</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>400</y>
        <width>171</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>FindFriend</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>181</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>AddFriend</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>400</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>DeleteFriend</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Help</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>ContactModerator</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <widget class="QPushButton" name="nextButton">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>55</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="prevButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>55</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
     <widget class="QLabel" name="messageLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>840</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="SubCommunityLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>840</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Historic</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="PositionLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>55</y>
        <width>660</width>
        <height>18</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTabs">
    <property name="title">
     <string>Tabs</string>
    </property>
    <addaction name="showcases"/>
    <addaction name="action_Fish"/>
    <addaction name="action_Equipment"/>
    <addaction name="action_Peripherals"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuProfile">
    <property name="title">
     <string>Profile</string>
    </property>
    <addaction name="actionCommunities"/>
    <addaction name="action_Friends"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Help"/>
   </widget>
   <addaction name="menuTabs"/>
   <addaction name="menuProfile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="showcases">
   <property name="text">
    <string>&amp;Showcases</string>
   </property>
  </action>
  <action name="action_Fish">
   <property name="text">
    <string>&amp;Fish</string>
   </property>
  </action>
  <action name="action_Equipment">
   <property name="text">
    <string>&amp;Equipment</string>
   </property>
  </action>
  <action name="action_Peripherals">
   <property name="text">
    <string>&amp;Peripherals</string>
   </property>
  </action>
  <action name="actionCommunities">
   <property name="text">
    <string>&amp;Communities</string>
   </property>
  </action>
  <action name="action_Friends">
   <property name="text">
    <string>&amp;Friends</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
